### DISCUSSION TITLE
Redesigning the header

### DISCUSSION THREAD

**Discussion by kevinmahrous** [msk-apps/msk-markdown#24] [Ideas]
Stuff about redesigning header goes here in order to be clear and clean.

**Comment by lee-jongwoo** [comment]
I created some blueprins with sketch.

1. Menu Bar Design
![Proto](https://user-images.githubusercontent.com/70620386/135833560-954c911a-d076-4882-8bc1-0d3d8e46e513.jpg)

2. Ribbon Menu design
![Proto 2](https://user-images.githubusercontent.com/70620386/135833572-892fc92f-0404-479c-8dcd-150e7c4469af.jpg)

**Reply by kevinmahrous** [reply]
I liked the second one but where's the "Line 1 Col 0" or because this's just a design? Thanks!

**Reply by lee-jongwoo** [reply]
@kevinmmansour Maybe put it at the right corner together?
(The setting and about buttons are gone)
It's just a design not the real one, there's plenty of room for improvement.
The first one also has some advantages, like doing stuff with less clicks :)

The first one uses dropdowns in its menu bar, so stuff like "open" and "save" goes under "file".
The second one uses ribbon menus, so buttons are divided into submenus, much like MS word.

**Reply by kevinmahrous** [reply]
Very nice, since the Settings and About buttons are gone where we'll fire the settings and about modal? By the way, in the second example, are they under the "Settings" tab?

I liked the second example. I'm just not sure about "The first one also has some advantages, like doing stuff with less clicks", I think the second one looks like MS word, wouldn't that be a bit productive or just time wasting for user?

**Reply by lee-jongwoo** [reply]
We can use the “MSK Markdown” logo to trigger the about modal, and there’s a settings button on the menu bar.
And yeah, I think the MS Word look will be more familiar to the user, too.

**Reply by kevinmahrous** [reply]
OK. Agree, we can start working on the second one.

I think we support the [Basic syntax](https://www.markdownguide.org/basic-syntax/) sheet and [Extended Syntax](https://www.markdownguide.org/extended-syntax/) sheet, no? Firstly, we should complete the missing markdown stuff, like "Footnotes", etc. Then redesign the header and add a button for every markdown element and the corresponding JS for that as well as some styling for some elements that we don't support yet like "Footnotes" and that would be brilliant work and we'll be ready to go.

But that'll take a lot of time, really a lot of imagination. :)

**Reply by LaideLawal** [reply]
Hello, I am interested in @lee-jongwoo suggestion and I agree with the house that the second design is nice and cool. I would be glad if you can permit me to give the design a shot. Thank you
